docs/ops/normalization/MVN_6.md

```diff
@@ -0,0 +1,98 @@
+## MVN <a name="MVN"></a> {#openvino_docs_ops_normalization_MVN_6}
+
+**Versioned name**: *MVN-6*
+
+**Category**: *Normalization*
+
+**Short description**: Calculates mean-variance normalization of the input tensor.
+
+**Detailed description**
+
+*MVN* subtracts mean value from the input blob:
+\f[
+o_{i} = i_{i} - ReduceMean(i_{k}, axes)
+\f]
+
+If *normalize_variance* is set to `true`, the output blob is divided by variance. When normalizing the value, the number `eps` is added to the variance to avoid division by zero. According to the `eps_mode` flag's value, `eps` is added inside or outside the sqrt:
+
+* If `eps_mode` is `inside_sqrt`:
+\f[
+o_{i}=\frac{o_{i}}{\sum \sqrt {o_{k}^2+\epsilon}}
```

Comment:
I think you have mixed up positions of \sum and \sqrt

Reply:
Agree, the formula was taken from MVN previous version